--- patches/pdfjs-dist-original/legacy/build/pdf.mjs	1985-10-26 16:15:00
+++ node_modules/pdfjs-dist/legacy/build/pdf.mjs	2026-08-17 15:49:06
@@ -22541,10 +22541,7 @@
       if (this.#mainThreadWorkerMessageHandler) {
         return this.#mainThreadWorkerMessageHandler;
       }
-      const worker = await import(
-      /*webpackIgnore: true*/
-      /*@vite-ignore*/
-      this.workerSrc);
+      const worker = await import('pdfjs-dist/legacy/build/pdf.worker.min.mjs');
       return worker.WorkerMessageHandler;
     };
     return shadow(this, "_setupFakeWorkerGlobal", loader());
